{{cookiecutter.project_slug}}/config/asgi.py

```diff
@@ -0,0 +1,36 @@
+"""
+ASGI config for {{ cookiecutter.project_name }} project.
+
+It exposes the ASGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/dev/howto/deployment/asgi/
+
+"""
+import sys
+from pathlib import Path
+
+from django.core.asgi import get_asgi_application
+
+# This allows easy placement of apps within the interior
+# {{ cookiecutter.project_slug }} directory.
+app_path = Path(__file__).parents[1].resolve()
+sys.path.append(str(app_path / "{{ cookiecutter.project_slug }}"))
+
+# This application object is used by any ASGI server configured to use this file.
+django_application = get_asgi_application()
+# Apply ASGI middleware here.
+# from helloworld.asgi import HelloWorldApplication
+# application = HelloWorldApplication(application)
```

Comment:
if `use_sentry` and `use_async` are enabled, [SentryAsgiMiddleware](https://docs.sentry.io/platforms/python/asgi/) needs to be used / applied. I can add this if needed. 👏

Reply:
Good point, I don't mind merging it wihtout, we can add it in a further PR I think.